docs/source/tensor_view.rst

```diff
@@ -73,6 +73,7 @@ For reference, here’s a full list of view ops in PyTorch:
 - :meth:`~torch.Tensor.unbind`
 - :meth:`~torch.Tensor.split`
 - :meth:`~torch.Tensor.split_with_sizes`
+- :meth:`~torch.Tensor.swapdim`
```

Comment:
Also add entries to tensor.rst, since swapdims and swapaxes will be methods, too